### ISSUE TITLE
[Bug] Item-button-right class is broken (position+dimension)

### ISSUE THREAD

**Issue by fredgalvao** [driftyco/ionic#1327]
(I've tested this on 0.9.27, beta1, beta2, beta3, beta4, and nightly{2042})
(I posted this [on the forums](http://forum.ionicframework.com/t/item-button-right-class-is-broken-position-dimension/3671) but got no response in 7 days, so I'm creating it here)

I'm trying to make a list that has a button visible at all times, and two other optionButtons bellow accessible through swipe. I didn't have the ionOptionButtons until now because I was waiting on the change of the syntax for the ionList to be refactored (as it was recently, **thanks!**). And now that I'm starting to implementing it, I had to change from

```
<div class="list">
  <div class="item item-button-right" ng-repeat="item in itemList">
    <h2>Text</h2>
    <p>Moar Text</p>
    <button class="button button-assertive">Inline button</button>
  </div>
</div>
```

to this:

```
<ion-list>
  <ion-item class="item item-button-right" ng-repeat="item in itemList">
    <h2>Text</h2>
    <p>Moar Text</p>
    <button class="button button-assertive">Inline button</button>
    <ion-option-button class="button button-assertive">Option button</ion-option-button>
  </ion-item>
</ion-list>
```

because swipe functionality doesn't come on pure class markup. While doing that, I found out that the div structure that ion-item creates at the end doesn't work with **item-button-right**. The space that the main body of the item creates at it's right is left empty, and the button (that should be using that space on the right) is at a strange position. This causes another issue with the swipe gesture and the appearance of the ionOptionButton: as soon as I start swipping, the ionOptionButton is completely visible, occupying the space on the right that was created by the main item div (the one I mentioned couple of lines above).

It's important to note that the issue with **ionOptionButton** is NOT what causes the issue with the **item-button-right class**, it's actually the other way around. You can comment out the ionOptionButton on the codepen example to see that the item-button-right still messes up.

http://codepen.io/fredgalvao/pen/Eziwf

**Comment by fpolimen** [comment]
I still see the same problem in the current release,
Looking at the codepen provided above (http://codepen.io/fredgalvao/pen/Eziwf) and
if I delete the ionOptionButton I see that the item-button-right positions correctly
instead 
if I comment out () it I still see the problem on the button-right

I have the same issue in my local system using chrome.
